2022 - LA - World Eaters.cat

```diff
@@ -3255,19 +3255,24 @@
                 <cost name="Pts" typeId="d2ee-04cb-5f8a-2642" value="15"/>
               </costs>
             </selectionEntry>
-            <selectionEntry id="34ff-115f-b2ad-9c40" name="Ravager w/ Heavy Weapon (1 in every 5)" hidden="false" collective="false" import="true" type="model">
+            <selectionEntry id="34ff-115f-b2ad-9c40" name="Ravager w/ Rad Missile Launcher (1 in every 5)" hidden="false" collective="false" import="true" type="model">
```

Comment:
Because these guys melee choice isn't collective they don't group as collective. Also, their "two blades" option is collective so that causes some weirdness. 

Finally, should we regen the ID on the heavy weapon guy that turned into rad missile launcher so someone who had a list with these before doesn't find their thunder hammers all turned into missile launchers?